src/main/java/logic/modulelogic/TimetableParser.java

```diff
@@ -0,0 +1,94 @@
+package logic.modulelogic;
```

Comment:
Should the explanation for the sequence diagram for `TimetableParser` in the DG have more elaboration and links to the diagram? I felt that there was not enough explanation for it.

On the other hand, the explanations for the other 2 diagrams were rather well linked to the sequence diagram.